view/frontend/web/internals/common.js

```diff
@@ -834,3 +834,20 @@ var AlgoliaBase64 = {
 		return string;
 	}
 };
+
+var htmlspecialchars_decode = function(string) {
+	var unescapedString = string,
+		specialchars = [
+			[ '"', '&quot;' ],
+			[ '>', '&gt;' ],
+			[ '<', '&lt;' ],
+			[ '&', '&amp;' ]
```

Comment:
it is usually expected to include `'`. Look for example in this lodash implementation: https://github.com/lodash/lodash/blob/master/unescape.js#L3-L7